### ISSUE TITLE
Training setting?

### ISSUE THREAD

**Issue by feifei-Liu** [pairlab/Adapt3R#2]
Thank you for providing such a solid job. I really hope to reproduce your work. After data processing, "python scripts/process _ libero _ data.py task = libero _ 90 _ data", we get the libero_90_processed folder. However, we noticed that libero_90 was used in the training, so we take libero_90_processed as libero_90.

Strangely, the loss we trained never converges, whether the decoder be of ACT or Difficulty Policy.

![Image](https://github.com/user-attachments/assets/93f5c94b-273c-46f8-ab85-2c9a3ff4f23b)

This is our training script. 
python scripts/train.py \
    --config-name=train.yaml \
    task=libero_90_hybrid \
    algo=act \
    algo/encoder=adapt3r \
    algo.chunk_size=15 \
    exp_name=EXP_NAME \
    variant_name=VARIANT_NAME \
    seed=0

Where task=libero_90_hybrid, I don't quite understand the meaning of hybrid. Can you provide some training logs for our training reference? 

Thank you very much!

**Comment by feifei-Liu** [comment]
![Image](https://github.com/user-attachments/assets/24180823-639e-4c87-a5d3-98f884ffc982)

**Comment by albertwilcox** [comment]
Hi, thanks for pointing out the bug. I'll try and reproduce your issues and then push a fix ASAP.

**Comment by albertwilcox** [comment]
Thanks again for the interest in my code base. Unfortunately I wasn't able to reproduce the issue with nans. Could you please run with anomaly detection and let me know where the nan arises?

**Comment by feifei-Liu** [comment]
Hi，
Thank you to help me！I notice that is the bug of "noise_scheduler.add_noise". 
https://github.com/pairlab/Adapt3R/blob/main/adapt3r/algos/diffusion_policy.py#L102

![Image](https://github.com/user-attachments/assets/633e560c-a303-4a79-b451-df7c55819e79)

After the ”add_noise", the action noise has NAN:

![Image](https://github.com/user-attachments/assets/1bf9b706-1641-42ca-b722-6dc2b9235ff5)

Then the noise pred all is NAN.

**Comment by albertwilcox** [comment]
Just pushed a fix! There was a subtle bug in the normalization code that I fixed. Please let me know if you have other issues and thanks again for using our code.

**Comment by feifei-Liu** [comment]
Thank you for your reply. After fixing the problem of normlization, the model can converge. Can you support the training of multiple gpus on single machine? At present, the training of single gpu is very time-consuming.
